lq1/default.cfg

```diff
@@ -82,19 +105,70 @@ sensitivity 3
+alias 	"fullammo"	    "impulse 9"
+alias 	"quad"		    "impulse 255"
+
+// Modern style melee attackcon_nextweapon
+alias   "+melee"        "impulse 1;+attack"
+alias   "-melee"        "-attack"
+
+// Zoom aliases
+alias   "+lq-zoom90"    "fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fov 40"
+alias   "-lq-zoom90"    "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90"
+alias   "+lq-zoom120"   "fov 110;wait;fov 100;wait;fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fov 40"
+alias   "-lq-zoom120"   "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90;wait;fov 100;wait;fov 110;wait;fov 120"
+
+// Dpad doom style weapon switch
+alias "+con_meleeatk"   "impulse 1;+attack;wait;bind LEFTARROW con_explguns0;bind RIGHTARROW con_nailguns0;bind UPARROW con_shotguns0"
+alias "-con_meleeatk"   "-attack"
+alias "con_shotguns0"   "impulse 2;wait;bind UPARROW con_shotguns1;wait;echo;echo;echo;echo;bind LEFTARROW con_explguns0;bind RIGHTARROW con_nailguns0"
+alias "con_shotguns1"   "impulse 2;wait;impulse 3;wait;bind UPARROW con_shotguns0;wait;echo;echo;echo;echo;bind LEFTARROW con_explguns0;bind RIGHTARROW con_nailguns0"
+alias "con_nailguns0"   "impulse 4;wait;bind RIGHTARROW con_nailguns1;wait;echo;echo;echo;echo;bind LEFTARROW con_explguns0;bind UPARROW con_shotguns0"
+alias "con_nailguns1"   "impulse 4;wait;impulse 5;wait;bind RIGHTARROW con_nailguns0;wait;echo;echo;echo;echo;bind LEFTARROW con_explguns0;bind UPARROW con_shotguns0"
+alias "con_explguns0"   "impulse 6;wait;bind LEFTARROW con_explguns1;wait;echo;echo;echo;echo;bind RIGHTARROW con_nailguns0;bind UPARROW con_shotguns0"
+alias "con_explguns1"   "impulse 6;wait;impulse 7;wait;bind LEFTARROW con_explguns0;wait;echo;echo;echo;echo;bind RIGHTARROW con_nailguns0;bind UPARROW con_shotguns0"
+alias "con_lightgun0"   "impulse 8;wait;echo;echo;echo;echo;bind LEFTARROW con_explguns0;bind RIGHTARROW con_nailguns0;bind UPARROW con_shotguns0"
+alias "con_nextweapn"   "impulse 10;wait;bind LEFTARROW con_explguns0;bind RIGHTARROW con_nailguns0;bind UPARROW con_shotguns0"
+alias "con_prevweapn"   "impulse 12;wait;bind LEFTARROW con_explguns0;bind RIGHTARROW con_nailguns0;bind UPARROW con_shotguns0"
+
+// Keyboard doom style weapon switch
+alias "doom_weapon_switch_info" "echo; echo; echo; echo --- ; echo doom_weapon_switch is a number row weapon switch style which is more like doom and halflife ; echo 1 is axe;echo 2 is Shotgun and Super Shotgun ; echo 3 is Nailgun and Super Nailgun ; echo 4 is Grenade Launcher and Rocket Launcher ; echo 5 is Lightning Gun; echo ---- ;echo type doom_weapon_switch1 to enable it ; echo doom_weapon_switch0 to revert it "
+
+alias "doom_weapon_switch1" "echo doom style weapon switch enabled! ; bind MOUSE5 +key_meleeatk ; bind 1 key_meleewep0 ; bind 2 key_shotguns0 ; bind 3 key_nailguns0 ; bind 4 key_explguns0 ; bind 5 con_lightgun0 ; unbind 6 ; unbind 7 ; unbind 8 ; bind MWHEELUP key_nextweapn ; bind MWHEELDOWN key_prevweapn"

```

Comment:
It feels like it would be clean if this was an `exec doomstyle_switch.cfg` that would also contain the necessary aliases etc. Though I'm not sure on that. Ignore if you feel this is irrelevant.

Reply:
i dont see the point for separating it from the the controller binds